### ISSUE TITLE
Regression between version 1.2.2 and 1.2.3/1.2.4

### ISSUE THREAD

**Issue by gwenn** [bytedeco/javacpp#137]
With the [javacpp branch](https://github.com/gwenn/sqlite-jna/tree/javacpp) of `sqlite-jna`:

```
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2169:17: error: functions that differ only in their return type cannot be overloaded
JNIEXPORT void* JavaCPP_org_sqlite_SQLite_00024AggregateFinalCallback_allocate_callback(::sqlite3_context* arg0) {
          ~~~~~ ^
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2108:16: note: previous definition is here
JNIEXPORT void JavaCPP_org_sqlite_SQLite_00024AggregateFinalCallback_allocate_callback(::sqlite3_context* arg0) {
          ~~~~ ^
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2170:12: error: cannot initialize return object of type 'void *' with an rvalue of type 'void'
    return JavaCPP_org_sqlite_SQLite_00024AggregateFinalCallback_allocate_callback_instance(arg0);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2190:12: error: cannot initialize return object of type 'void *' with an rvalue of type 'void'
    return JavaCPP_org_sqlite_SQLite_00024AggregateFinalCallback_getAggregateContext_callback_instance(arg0);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2357:17: error: functions that differ only in their return type cannot be overloaded
JNIEXPORT void* JavaCPP_org_sqlite_SQLite_00024ScalarCallback_allocate_callback(::sqlite3_context* arg0, int arg1) {
          ~~~~~ ^
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2284:16: note: previous definition is here
JNIEXPORT void JavaCPP_org_sqlite_SQLite_00024ScalarCallback_allocate_callback(::sqlite3_context* arg0, int arg1, sqlite3_value** arg2) {
          ~~~~ ^
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2358:12: error: no matching function for call to object of type 'JavaCPP_org_sqlite_SQLite_00024ScalarCallback'
    return JavaCPP_org_sqlite_SQLite_00024ScalarCallback_allocate_callback_instance(arg0, arg1);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2287:53: note: candidate function not viable: requires 3 arguments, but 2 were provided
void JavaCPP_org_sqlite_SQLite_00024ScalarCallback::operator()(::sqlite3_context* arg0, int arg1, sqlite3_value** arg2) {
                                                    ^
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2378:12: error: no matching function for call to object of type 'JavaCPP_org_sqlite_SQLite_00024ScalarCallback'
    return JavaCPP_org_sqlite_SQLite_00024ScalarCallback_getAuxData_callback_instance(arg0, arg1);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/gwen/Java/sqlite-javacpp/target/classes/org/sqlite/jniSQLite.cpp:2287:53: note: candidate function not viable: requires 3 arguments, but 2 were provided
void JavaCPP_org_sqlite_SQLite_00024ScalarCallback::operator()(::sqlite3_context* arg0, int arg1, sqlite3_value** arg2) {
                                                    ^
```

**Comment by saudet** [comment]
Could pinpoint the commit that caused the behavior to change?

BTW, it would help if you could created presets so I can more easily make sure that I'm not breaking anything for SQLite:
https://github.com/bytedeco/javacpp-presets/wiki/Create-New-Presets

**Comment by gwenn** [comment]
It seems to be this commit: https://github.com/bytedeco/javacpp/commit/1380e19f51dd12b7083356e3601f6b5fc763da35
And I skip the parser.

And sorry, but I would prefer not to create presets.

**Comment by saudet** [comment]
Fixed! Thanks for reporting.

**Comment by saudet** [comment]
Fix is in version 1.2.5! Thanks again for reporting
